tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -4165,77 +4201,30 @@ <h2>Show similarity to selected datapoint</h2>
           // back to custom (default).
           this.resetOptimizationSelected_();
 
-          const feature1 = this.selectedBreakdownFeature;
-          if (feature1 == '') {
-            this.selectedSecondBreakdownFeature = '';
-          }
```

Comment:
probably still want this logic here, which clears out the secondary slice selection when the primary slice selection is cleared. otherwise if you have both slices set, then clear the primary, the view goes back to showing only overall stats, but then when you select a slice again, the secondary slice is already selected as whatever it was last and you don't just see the primary slice you selected.